runtime/src/lib.rs

```diff
@@ -735,7 +734,7 @@ construct_runtime!(
         CouncilElection: election::{Module, Call, Storage, Event<T>, Config<T>},
         Council: council::{Module, Call, Storage, Event<T>, Config<T>},
         Memo: memo::{Module, Call, Storage, Event<T>},
-        Members: members::{Module, Call, Storage, Event<T>, Config<T>},
+        Members: membership::{Module, Call, Storage, Event<T>, Config<T>},
```

Comment:
headsup @Lezek123 all `api.query.members.*` and `api.query.tx.members.*` in polkadot-js apps will need to be updated to `api.*.membership.*` after this change.

Reply:
It seems like the `polkadot/api` still sees it as `members`